Recrutify/Recrutify.Host/Constants.cs

```diff
@@ -11,7 +13,12 @@ public static class Policies
 
         public static class Cors
         {
-            public const string CorsForUI = "CorsForUI";
+            public const string CorsForUI = nameof(CorsForUI);
+        }
+
+        public static class Roles
+        {
+            public const string ProjectRoles = nameof(ProjectRoles);
```

Comment:
cause you use Constants.Roles.ProjectRoles.ToLower() you can just write ProjectRoles = "projectRoles";
we use nameof(ProjectRoles) when constant value always same as constant name